files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -173,6 +173,14 @@
     - 'root'
     - '*'
 
+- lineinfile:
+    create: yes
+    path: /etc/security/limits.conf
```

Comment:
Rather than modifying the existing file, can we instead add a new file in `/etc/security/limits.d` to do this? If so, then we can add that file to the repository like this:
```
± cat files/common/etc/security/limits.d/delphix.conf
root soft nofile 32768
delphix soft nofile 32768
```

I'm not sure if files under `limits.d` will take effect, but if they do, then I think this would be a better approach; e.g. less chance to conflict with upstream changes to the default `limits.conf` file, and easier to change or remove these settings on upgrade.

Reply:
Although.. now I see we already modify `limits.conf` just a few lines above.. so I'll go ahead and approve this, since it's consistent with our existing logic..

I still think both should probably be moved to a new file under `limits.d`, but we can take that on in a follow up change.

Reply:
https://delphix.atlassian.net/browse/DLPX-78757 has been created to track this improvement.